### ISSUE TITLE
Preserve page-based text output of pdf2htmlEX documents

### ISSUE THREAD

**Issue by sathomas** [oaeproject/Hilary#1102]
For the front-end issue https://github.com/oaeproject/3akai-ux/issues/3911 we should preserve the pure text versions of pdf2htmlEX documents and make them available to the front-end for screen reader support. The text documents should be broken up by pages and stored with the HTML and CSS content.

**Comment by stuartf** [comment]
This was incorporated in #1103
